src/docs/product/sentry-basics/tracing/distributed-tracing.mdx

```diff
@@ -117,7 +117,7 @@ Now, for the sake of completeness, back to our spans:
   - 1 span for the authentication query
   - 1 span for the query retrieving data
 
-To wrap up the example: after instrumenting all of your services, you might discover that - for some reason - it's the auth query in your database server that is making things slow, accounting for more than half of the time it takes for your entire page load process to complete. Tracing can't tell you _why_ that's happening, but at least now you know where to look!
+To wrap up the example: after instrumenting all of your services, you might discover that - for some reason - it's the auth query in your database server that is making things slow, accounting for more than half of the time it takes for your entire page load process to complete. Tracing can't tell you _why_ that's happening, but at least now you know where to look.
```

Comment:
```suggestion
To wrap up the example: after instrumenting all of your services, you might discover that - for some reason - it's the auth query in your database server that is making things slow, accounting for more than half of the time it takes for your entire page load process to complete. Tracing won't be able to tell you _why_ that's happening, but it will show you where to look.
```